app/src/main/java/com/github/se/wanderpals/ui/screens/trip/Map.kt

```diff
@@ -191,42 +264,122 @@ fun Map(mapViewModel: MapViewModel, client: PlacesClient) {
+          Column {
+            if (placeName.isNotEmpty()) {
+              Text(
+                  modifier = Modifier.align(Alignment.CenterHorizontally).padding(bottom = 16.dp),
+                  text = placeName)
+            }
+            if (placeBusinessStatus.isNotEmpty()) {
+              Row(modifier = Modifier.padding(bottom = 8.dp)) {
+                Icon(
+                    imageVector = Icons.Default.Info,
+                    contentDescription = "Business Status Icon",
+                    modifier = Modifier.size(24.dp))
+                Text(text = placeBusinessStatus)
+              }
+            }
+            if (placeAddress.isNotEmpty()) {
+              Row(modifier = Modifier.padding(bottom = 8.dp)) {
+                Icon(
+                    imageVector = Icons.Default.Place,
+                    contentDescription = "Place Icon",
+                    modifier = Modifier.size(24.dp))
+                Text(text = placeAddress)
+              }
+            }
+            if (placeRating.isNotEmpty()) {
+
+              Row(modifier = Modifier.padding(bottom = 8.dp)) {
+                Icon(
+                    imageVector = Icons.Default.Star,
+                    contentDescription = "Rating Icon",
+                    modifier = Modifier.size(24.dp))
+                Text(text = "$placeRating/5.0")
+              }
+            }
+            if (placeUserRatingsTotal.isNotEmpty()) {
+              Row(modifier = Modifier.padding(bottom = 8.dp)) {
+                Icon(
+                    imageVector = Icons.Default.Person,
+                    contentDescription = "Total Icon",
+                    modifier = Modifier.size(24.dp))
+                Text(text = placeUserRatingsTotal)
+              }
+            }
+            if (placePhoneNumber.isNotEmpty()) {
+              Row(modifier = Modifier.padding(bottom = 8.dp)) {
+                Icon(
+                    imageVector = Icons.Default.Phone,
+                    contentDescription = "Phone Icon",
+                    modifier = Modifier.size(24.dp))
+                Text(text = placePhoneNumber)
+              }
+            }
+            if (placeWebsite.isNotEmpty()) {
+              Row(modifier = Modifier.padding(bottom = 8.dp)) {
+                Text(text = "Visit Website: ")
+                Icon(
+                    imageVector = Icons.Default.Search,
+                    contentDescription = "Phone Icon",
+                    modifier = Modifier.size(24.dp).clickable { uriHandler.openUri(placeWebsite) })
+              }
+            }
+
+            if (placeOpeningHours.isNotEmpty()) {
+              Spacer(modifier = Modifier.height(16.dp))
+              val listOfDays = placeOpeningHours.removePrefix("[").removeSuffix("]").split(", ")
+              listOfDays.forEach {
+                Text(
+                    modifier = Modifier.align(Alignment.CenterHorizontally).padding(bottom = 8.dp),
+                    text = it)
```

Comment:
Check that "it" isn't null, I don't know why but "null" can be displayed:
![image](https://github.com/WanderPals/WanderPals/assets/80963196/e28f4cd5-2bfe-434d-a93e-0f6ce2d276b4)

Reply:
Okay, I will fix it